syntax-highlight/vim/_vimrc

```diff
@@ -0,0 +1,15 @@
+" Put this in the final lines of the file.
+
+:syntax on
+autocmd BufRead,BufNewFile *.mya set filetype=mya
+
+augroup MyaHighlight
+  autocmd!
+  autocmd FileType mya highlight MyaDeclaration guifg=#569CD6 ctermfg=Blue
+  autocmd FileType mya highlight MyaOperator guifg=#D4D4D4 ctermfg=White
+  autocmd FileType mya highlight MyaNumber guifg=#B5CEA8 ctermfg=Cyan
+  autocmd FileType mya highlight MyaString guifg=#CE9178 ctermfg=Green
```

Comment:
I noticed you are using the same color on strings  and comments. Could you please use another color for strings? Like `Magenta` for example, or another color you think it's fit good.